ecs_fargate/README.md

```diff
@@ -160,6 +160,10 @@ Datadog's default CloudWatch crawler polls metrics once every 10 minutes. If you
 
 ### Log Collection
 
```

Comment:
can we mention that FireLens is the preferred option among the two?

Reply:
made that one first and added an introductory sentence